exercises/strings/rotN/rotN.js

```diff
@@ -15,15 +15,37 @@
  * @returns {string} A lowercase copy of the input string
  */
 
-function rotN(string) {
+function rotN(string, n) {
   // This is your job. :)
+  var i;
+  var new_string = '';  
```

Comment:
The convention in JavaScript is to name variables/functions/etc with "camel case", like `newString`. The convention you're using is called "snake case", like `new_string`. I personally _prefer_ snake case,  but it's important to have a "when in Rome" attitude if you other folks in the ecosystem to be able to help you out.

For example, if you submitted code to an open source JS project using snake case, it'd probably get rejected on those grounds.

Every language community has its own conventions and you just have to learn how to follow them. 🤷